app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -125,15 +125,61 @@ class PatientSummaryScreenController @Inject constructor(
         .map { (_, patientUuid) -> { ui: Ui -> ui.showUpdatePrescribedDrugsScreen(patientUuid) } }
   }
 
-  private fun handleBackClicks(events: Observable<UiEvent>): Observable<UiChange> {
-    return events
+  private fun backAndDoneClicks(events: Observable<UiEvent>): Observable<UiChange> {
```

Comment:
This is another thing that would have been covered already if we had a guidelines wiki in page. We've so far named ui change functions according to the action they're performing and not the events they're consuming. Can you rename both `backAndDoneClicks` and `scheduleAppointmentSheetCloses`?